packages/terra-data-grid/src/subcomponents/ColumnHeaderCell.module.scss

```diff
@@ -23,6 +23,11 @@
       height: 100%;
       padding: var(--terra-data-grid-header-cell-padding, 5px);
       width: 100%;
+
+      &:focus {
+        outline: var(--terra-data-grid-cell-focus-outline, 3px dashed #000);
+        outline-offset: var(--terra-data-grid-cell-focus-outline-offset, -2px);
+      }
```

Comment:
Don't we already have this styling for the DataGrid as a whole?  I know that it existed for the WorklistDataGrid.  We should not have to duplicate the style, correct?

Reply:
Removed